src/app/components/wrapper-components/demo-card-wrapper/demo-card-wrapper.component.spec.ts

```diff
@@ -74,11 +82,57 @@ describe('DemoCardComponent', () => {
         // Then it should not call rules.choose
         expect(testUtils.getComponent().gameComponent.rules.choose).not.toHaveBeenCalled();
     }));
+
     it('should do nothing when you pass', fakeAsync(async() => {
         // Given any starting state of component
         // When passing
         const result: void = await testUtils.getComponent().onCancelMove('not even necessary');
         // Then nothing should have happend (for coverage sake)
         expect(result).withContext('should be null').toBe();
     }));
+
+    it('should reload node when input are updated by parents', fakeAsync(async() => {
```

Comment:
inputs* car "are", mais c'est quoi les inputs ? C'est plutôt "when another component is loaded" right ?